test/data/error_match.json

```diff
@@ -1,14 +1,18 @@
 {
-  "biome_err": [
-    "Key biome: Metadata value at key value does not have envo_ontology ancestor term ENVO:00000428"
-  ],
-  "material_err": [
-    "Key ENIGMA:material: Metadata value at key value does not have envo_ontology ancestor term ENVO:00010483"
-  ],
-  "fear_err": [
-    "Key feature: Metadata value at key value does not have envo_ontology ancestor term ENVO:01000813"
-  ],
-  "outta_range": [
-    "Key latitude: Metadata value at key value is not within the range [-90.0, 90.0]"
-  ]
+  "biome_err": {
```

Comment:
what do the errors look like if there are multiple errors per row?

Reply:
The errors are not grouped in any way at the moment. Just an array of `SampleContentError`s *. The above is just a structure of expected test values (each sample for that test only has one expected error), it isn't symmetric to the error format. I would usually just put the above in the test itself, but the existing test had used the JSON file so I rolled with it. Could refactor it. What do you think?

*This was done to maybe allow full-row and full-column errors in the future by leaving column or row as None, respectively. Right now, all SampleContentError have both row and column